src/Components/Components/src/HttpClientJsonExtensions.cs

```diff
@@ -13,6 +13,11 @@ namespace Microsoft.AspNetCore.Components
     /// </summary>
     public static class HttpClientJsonExtensions
     {
+        private static readonly JsonSerializerOptions HttpClientSerializationOptions = new JsonSerializerOptions
```

Comment:
Spicy thought - @SteveSandersonMS - do we have somewhere specific to client-side blazor to put these code?

Reply:
Which code? All of the `HttpClientJsonExtensions`? Not really because it's mean to be part of the programming model that works in either place (e.g., when sharing components across environments).

Please let me know if you meant something different.